### ISSUE TITLE
Can't stop craftin!

### ISSUE THREAD

**Issue by Midaychi** [CleverRaven/Cataclysm-DDA#11195]
I'm actually not sure how to reproduce this normally; it was just a situation I managed to run into.
In this strange situation, I'm just unable to stop crafting if I do a specific thing. I get stuck in an endless loop. (and yes I do have the maximum trait debug cranked up for this run because I wanted to try something silly, but probably not relevant to this issue)

https://www.dropbox.com/s/9d7b1frjzi4s9by/Saint%20Martin.zip?dl=0

Load this save up, do nothing to alter the situation it's currently in (for the first test anyways). To reproduce the bug I'm talking about: Use the craft continuously binding to craft coffee into that glass bottle. After the fist success, it should tell you that it needs hotplate charges with the warning popup, but as soon as you dismiss it the process continues despite not having hotplate charges. I've found no way to exit this infinite loop, as num5 or period don't interrupt.

I specially prepared this save after discovering the issue so the hotplate only had three charges left, and the hotplate is the mess kit in the bed a tile to the left.

tested last on version c2c82c4

**Comment by nozedissafe** [comment]
I have experienced a similar bug (probably the same one, but a slight ambiguity in your post made me decide to leave this comment anyway). Situation: I was making some cured pelts into cured hides for tanning (never enough leather... never). I had five pelts in my inventory, and as soon as the craft continuously function had made those into hides, I got stuck between a debug message ("attempted a recipe with no available components", something like that) and the regular "component missing" message. Neither 5 nor ~ helped - hitting space when the debug message popped up only resulted in a black screen with the component missing message. If I hit enter on that screen I'd go back to the debug message. No recourse but to close the game. Kinda annoying.
